UPGRADING.md

```diff
@@ -9,6 +9,60 @@ The `rack_response` method has been deprecated and the `error_response` method h
 
 See [#2414](https://github.com/ruby-grape/grape/pull/2414) for more information.
 
+#### Change in parameters precedence
+
+*(only applicable if Grape::Extensions::Hash::ParamBuilder is used)*
```

Comment:
Say "When using together with `Grape::Extensions::Hash::ParamBuilder`, `route_param` takes ... , which now matches the default param bulider behavior."

Instead of original behavior say "This was a regression introduced by [link to PR] in Grape v1.8.0."

Reply:
Updated.